<launch>
	<node pkg="openfabmap2" type="run_node" name="run_node">
		<remap from="image" to="/camera/rgb/image_raw"/>
		<param name="vocab" value="$(find openfabmap2)/../../..//codebooks/new/vocab.yml"/>
		<param name="clTree" value="$(find openfabmap2)/../../..//codebooks/new/clTree.yml"/>
		<param name="trainbows" value="$(find openfabmap2)/../../..//codebooks/new/trainbows.yml"/>
		<param name="dumpFile" value="$(find openfabmap2)/../../..//codebooks/new/confusion_matrix.yml"/>
		<param name="DetectorType" value="SURF"/>
		<param name="FabMapVersion" value="FABMAP2"/>
		<param name="NewPlaceMethod" value="Sampled"/>
		<param name="sampleRate" value="2"/>
		<param name="visualise" value="true"/>
		<param name="ShowConfusionMatrix" value="true"/>
		<param name="minMatchValue" value="0.98"/>
		<param name="maxMatches" value="3"/>
		<param name="DisableSelfMatch" value="false"/>
		<param name="SelfMatchWindow" value="50"/>
		<param name="DisableUnknownMatch" value="true"/>
		<param name="AddOnlyNewPlaces" value="false"/>
	</node>
</launch>
